cmd/geth/main.go

```diff
@@ -260,6 +260,17 @@ func init() {
 	app.Flags = append(app.Flags, metricsFlags...)
 
 	app.Before = func(ctx *cli.Context) error {
+		// If flag is set by command line ignore config file
```

Comment:
That goes for every single item that exists both on command line  and toml. I don't see why handling verbosity needs this special handling?

Reply:
That's because verbosity is part of a set of flags not included in the read/write of the config file, and they are set at the very beginning of the program (the `init`). This is why we need to read the config file for the verbosity here, and then much later read it for the others flags. If we put the verbosity setting with other config flags read from file, there will be much part of the code that will run with the default verbosity (or the verbosity passed by command line) until the calling to the method `makeConfigNode`